utils/mil_tools/mil_ros_tools/online_bagger/config/online_bagger_example.yaml

```diff
@@ -1,15 +0,0 @@
-# list an arbitrary set of topics to  subscribe to and stream:
```

Comment:
I think this is useful to keep as an example in an example launch.
Idea:
Move `online_bagger.py` into `utils/mil_tools/nodes`, then making a `utils/mil_tools/launch` where you put the launch file with the config in there as params in the launch.